vcpkg.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Is there any option to move this platform-specific file to `util/`?

Reply:
Apparently not. Reconiguring cmake in Visual Studio after moving `vcpkg.json` (and cleaning the cache) causes an error that packages cannot be found (also the `Running vcpkg install` lines do not appear)

Reply:
That's unfortunate but understandable. Thanks. In that case the PR LGTM as-is.